lib/index.coffee

```diff
@@ -34,7 +34,7 @@ class TidyMarkdown
 
   run: ->
     editor = atom.workspace.getActiveTextEditor()
-    if editor.getGrammar().scopeName isnt 'source.gfm' then return
+    if editor.getGrammar().scopeName isnt 'source.gfm' and editor.getGrammar().scopeName isnt 'text.md' then return 
```

Comment:
`editor.getGrammar().scopeName not in ['source.gfm', 'text.md']` would make this faster. Also there is some trailing whitespace. If you're using atom, then this should have been removed automatically.